data_store/controllers/notify.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from notifications_python_client.errors import HTTPError
+from notifications_python_client.notifications import NotificationsAPIClient
+from werkzeug.datastructures import FileStorage
+
+from config import Config
+from data_store.controllers.load_functions import get_or_generate_submission_id
+from data_store.db.queries import get_programme_by_id_and_round
+from submit.main.fund import FundConfig
+from submit.utils import get_friendly_fund_type
```

Comment:
Thoughts on breaking this submit import? 
`get_friendly_fund_type` is not too hard to move.. perhaps.
`FundConfig` not too sure what it is, but I don't think it's a Submit only thing.

Reply:
Yes, data store shouldn't import from submit.